### PR TITLE
[CLIENT] Update Resource Request Forms

### PR DESCRIPTION
**Pull request by MichelineMedhat**

### CODE REVIEW

File: client/graphql/types/index.tsx [hunk 1]
```diff
@@ -1,7 +1,9 @@
-import gql from 'graphql-tag';
-import * as Urql from 'urql';
+import gql from "graphql-tag";
```
Comment (yousinix) [inline client/graphql/types/index.tsx]: regenerate and don't format

File: client/pages/resource-request/create.tsx [hunk 2]
```diff
@@ -23,32 +27,54 @@ const CreateResourceRequest: React.FC<{}> = () => {
...
     >
       <ResourceRequestForm
         action="Add"
-        managers={data?.managers as EmployeesProfiles[]}
+        me={meData?.me as EmployeesProfiles}
+        profileData={
+          employeesProfilesData?.employeesProfiles as Pick<
+            EmployeesProfiles,
+            "function" | "title"
+          >[]
```
Comment (yousinix) [inline client/pages/resource-request/create.tsx]: `as EmployeeProfiles[]` will work better

File: client/pages/resource-request/create.tsx [hunk 3]
```diff
@@ -23,32 +27,54 @@ const CreateResourceRequest: React.FC<{}> = () => {
...
+        }
+        managers={managersNamesData?.managers as EmployeesProfiles[]}
         initialValues={initialValues}
         onSubmit={async (values, { setErrors }) => {
+          values.status = "Open";
+          values.startDate =
+            values.startDate === "" ? formatDate(new Date()) : values.startDate;
+          values.endDate =
+            values.endDate === "" ? formatDate(new Date()) : values.endDate;
+
```
Comment (yousinix) [inline client/pages/resource-request/create.tsx]: Fix this from the datepicker

**Review by yousinix — CHANGES_REQUESTED** [review]

File: client/pages/resource-request/create.tsx (hunk 3 above)
Comment (MichelineMedhat) [reply client/pages/resource-request/create.tsx]: I tried it & it didn't work

**Review by yousinix — APPROVED** [review]

**Pull request merged**